solidity/contracts/Gravity.sol

```diff
@@ -9,6 +9,10 @@ import "./CosmosToken.sol";
 
 pragma experimental ABIEncoderV2;
```

Comment:
@cgdusek review the contract one more time, because I updated your changes a bit to make them be compiled

Reply:
Okay